setup/ci/gcb.md

```diff
@@ -66,6 +66,9 @@ on the status of your build.
 
 7. In the **Payload Constraints** field, you can enter any of the top-level fields from the
 [Build object documentation](https://cloud.google.com/cloud-build/docs/api/reference/rest/v1/projects.builds#resource-build)
-as the key, and a Java regular expression as the value. For example, to match on an image produced,
-you can use the key `images` and the value `gcr\.io/my-project-id/my-application:latest` (replacing
-`my-project-id` and `my-application` with appropriate values).
+as the key, and a Java regular expression as the value.
+
+8. In the **Expected Artifacts** field, you can add any build artifacts as expected artifacts. For example,
+if the build produces a docker image, you can add an expected artifact of type *Docker* with a value of
+`gcr.io/my-project-id/my-application` (replacing `my-project-id` and `my-application` with appropriate values). You
```

Comment:
I would call out that images can be referenced by `:tag` here too -- we submit artifacts for both `gcr.io/image` and `gcr.io/image:tag` with references to `gcr.io/image@sha256:abc...`